mewarpx/mewarpx/utils_store/parallel_util.py

```diff
@@ -1,15 +1,18 @@
 from mpi4py import MPI as mpi
 import numpy as np
 
-
-from mewarpx.mwxrun import mwxrun
-
-
 comm_world = mpi.COMM_WORLD
 
-def mpiallreduce(data=None, op=None, comm=None):
-    if op is None:
+def mpiallreduce(data=None, opstring=None, comm=None):
+    if opstring is None:
```

Comment:
Is there a reason to not just make the default value for `opstring="SUM"` and remove the `if opstring is None` block?